blog/_posts/2017-10-05-TeamGemini-Happy-ending-yet-not-the-end.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Happy Ending, yet not the end !
+layout: post
+created_at: Sun Oct 05 2017
+permalink: blog/2017-10-05-TeamGemini-Happy-ending-yet-not-the-end
+current: blog
+author: Akarsha and Ramya
+twitter: team_Gemini_
+categories:
+- student-posts
+---
+
+## Rgsoc and Us 
+
+What a roller coaster ride it has been! It is disheartening to realize how soon it has ended. RGSoC is one of the most beautiful communities we could ever be a part of at this stage. We really learnt so much from our fellows and the community members. The best thing we learnt was giving back; giving back to the community which taught you multiple things, giving back to the society by helping out and supporting more and more people get into tech. 
+
+## What RGSoC taught us
+
+We would not be able to thank RGSoC enough for all we learned through this summer. Let alone be the fact that we are now comfortable reading huge codebases (Yes, we got the opportunity to work with two amazing Projects :D). RGSoC gave us the visibility in college and other communities in our city. It opened so many new doors for us to explore and learn. Besides contributing to opensource we learned new tools and technologies and thus improving our skill-set.  It gave us motivation to be a part of the open-source communities and boosted our confidence of giving talks in their meetups. 
+
+At one such opensource meetup, [Akarsha](https://twitter.com/akarsha98) also met the co-founder of her early-stage startup ‘Invadex’ which she is going to represent in the Alpha program of Web-summit this November which is regarded as the **“The Largest Tech Conference in the World”**. We’ll also organize Django girls, Lisbon to help more girls fall in love with programming and open-source. And because of our exposure with Scikit-Image, [Ramya](https://twitter.com/notrandomatalll) took to research with the Image Analysis and Biometrics lab of our college. This is how Rgsoc helped us kickstart our careers and grow a lot as individuals. 
```

Comment:
Remove one the in this sentence: 'the **“The Largest Tech Conference in the World”**'. 

This 'took to research' looks bit odd. Rephrase it.